SimCalorimetry/EcalEBTrigPrimProducers/plugins/EcalEBTrigPrimAnalyzer.cc

```diff
@@ -261,7 +460,522 @@ EcalEBTrigPrimAnalyzer::analyze(const edm::Event& iEvent, const  edm::EventSetup
 	hAllRechitEt_ ->Fill(rhEt);
     }
   }
+
+
+
+  //  float Etsum=0;
+  std::vector<SimpleCluster> cluCollection;
+  std::vector<SimpleCaloHit2> myHits;
+  if ( recHits_ ) {    
+
+    myHits.clear();
+    for (unsigned int j=0;j<rechit_EB_col.product()->size();j++) {
+      const EBDetId & rhID=(*rechit_EB_col.product())[j].detid();
+      float eta =  theBarrelGeometry->getGeometry(rhID)->getPosition().eta();
+      float energy= (*rechit_EB_col.product())[j].energy();
+      float et= energy/cosh(eta);
+      if (energy <=0) continue;
+      if (debug_) std::cout << " my Hits energy " << energy << " et " << et << std::endl;
+      SimpleCaloHit2 hit(et);
+      hit.setPosition(GlobalVector( theBarrelGeometry->getGeometry(rhID)->getPosition().x(), 
+				    theBarrelGeometry->getGeometry(rhID)->getPosition().y(), 
+				    theBarrelGeometry->getGeometry(rhID)->getPosition().z()));
+
+
+
+      hit.setId(rhID);
+      myHits.push_back(hit);	  
+
+
+    }
+    if ( debug_) std::cout << " myHits collection size " << myHits.size() << std::endl;
+    for (unsigned int iRH=0;iRH<myHits.size();++iRH) {
+      if (debug_) std::cout << " RecHit Et " << myHits[iRH].et() << std::endl;
+    }    
+
+
+    /*
```

Comment:
Delete commented out code, or justify with a comment why you still need to keep it